homeassistant/components/iss/sensor.py

```diff
@@ -28,23 +30,32 @@ async def async_setup_entry(
     """Set up the sensor platform."""
     coordinator: DataUpdateCoordinator[IssData] = hass.data[DOMAIN]
 
-    name = entry.title
     show_on_map = entry.options.get(CONF_SHOW_ON_MAP, False)
 
-    async_add_entities([IssSensor(coordinator, name, show_on_map)])
+    async_add_entities([IssSensor(coordinator, entry, show_on_map)])
 
 
 class IssSensor(CoordinatorEntity[DataUpdateCoordinator[IssData]], SensorEntity):
     """Implementation of the ISS sensor."""
 
+    _attr_has_entity_name = True
+    _attr_name = None
+
     def __init__(
-        self, coordinator: DataUpdateCoordinator[IssData], name: str, show: bool
+        self,
+        coordinator: DataUpdateCoordinator[IssData],
+        entry: ConfigEntry,
+        show: bool,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
-        self._state = None
-        self._attr_name = name
+        self._attr_unique_id = f"{DOMAIN}_people"
```

Comment:
If we use the entry ID here, we avoid collisions in the future.

```suggestion
        self._attr_unique_id = f"{entry.entry_id}_people"
```